deepchem/models/tests/test_generalize.py

```diff
@@ -188,3 +189,93 @@ def model_builder(model_dir):
   #  scores = model.evaluate(test_dataset, [classification_metric])
   #  for score in scores[classification_metric.name]:
   #    assert score > .5
+
```

Comment:
Unit tests look good to me(though typically we only test for overfitting, without splitting and evaluating on test_dataset). Do you have any idea about the approximate time for going through these tests?